packages/docs/assets/colors.scss

```diff
@@ -0,0 +1,5 @@
+$active-blue: #2550C0;
```

Comment:
Styling as different frameworks - either split as separate task or finish in context of this MR.
![image](https://user-images.githubusercontent.com/5394573/98268554-b744d480-1f9d-11eb-8d55-a9a6febe4d39.png)